ffi/diplomat/wasm/wasm-demo/src/ts/app.ts

```diff
@@ -3,7 +3,6 @@ import * as fdf from './fixed-decimal';
 import * as dtf from './date-time';
 import * as seg from './segmenter';
 
-import '../scss/styles.scss';
 import 'bootstrap/js/dist/tab';
 import 'bootstrap/js/dist/dropdown';
 import 'bootstrap/js/dist/collapse';
```

Comment:
Issue: I think TypeScript still won't like these boostrap includes... but this PR is still a net improvement.